src/main/java/co/cask/gcp/spanner/sink/SpannerSink.java

```diff
@@ -68,6 +87,22 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   @Override
   public void prepareRun(BatchSinkContext context) {
     config.validate();
+
+    try {
+      Spanner spanner = SpannerUtil.getSpannerService(config.getServiceAccountFilePath(), config.getProject());
+      DatabaseId db = DatabaseId.of(config.getProject(), config.instance, config.database);
+      DatabaseClient dbClient = spanner.getDatabaseClient(db);
+      DatabaseAdminClient dbAdminClient = spanner.getDatabaseAdminClient();
+      // create database
+      Database database = getOrCreateDatabase(dbAdminClient);
+      // create table
+      createTableIfNotPresent(dbClient, database);
+      // Close spanner connection to flush all the changes.
+      spanner.close();
+    } catch (IOException e) {
+      throw new RuntimeException("Exception while trying to get Spanner service. ", e);
```

Comment:
what throws an IOException? Is it just SpannerUtil.getSpannerService()?

If not, this message needs to be changed based on what can throw the exception